analyses/survival-analysis/survival-analysis_subtypes.Rmd

```diff
@@ -99,40 +110,44 @@ This way we subset the metadata to 1 single primary sample per patient ID
 independent_primary <- readr::read_tsv(independent_primary_file)
 
 # select only independent primary bs_ids to get overall survival
-metadata <- metadata %>%
+metadata_ind_dna <- metadata %>%
   dplyr::filter(Kids_First_Biospecimen_ID %in%
-                  independent_primary$Kids_First_Biospecimen_ID)
+                  independent_primary$Kids_First_Biospecimen_ID) 
+
+# check whether any RNA pts not in DNA subset
+metadata_rna <- metadata %>%
+  filter(experimental_strategy == "RNA-Seq" & 
+           tumor_descriptor == "Initial CNS Tumor" & 
+           !is.na(OS_years)) %>%
+  unique()
+
+rna_pts_to_add <- setdiff(unique(metadata_rna$Kids_First_Participant_ID), metadata_ind_dna$Kids_First_Participant_ID)
```

Comment:
This line is a confusing - There is a `#nope` comment, but when I print this variable out (and it should be printed in the notebook if we need to see the results to get to that "nope" conclusion), it has output consisting of 5 patient ids:

```
> rna_pts_to_add <- setdiff(unique(metadata_rna$Kids_First_Participant_ID), metadata_ind_dna$Kids_First_Participant_ID)
> rna_pts_to_add
[1] "PT_63HDZHCN" "PT_9BZETM0M" "PT_0SPKM4S8" "PT_0MXPTTM3" "PT_S0Q27J13"
```

Based on this seems to me there _are_ RNA patients not in the DNA subset?

Reply:
I thought these were not primary tumors or maybe not RNA-only samples, but I can check again and should update the code for this.

Reply:
hmm, I still get `character(0)` - printing and will commit

Reply:
Hm, let me see again..